src/main.rs

```diff
@@ -1,18 +1,23 @@
+use anyhow::{Context, Error};
 use serde::Deserialize;
 
-fn main() {
+fn main() -> Result<(), Error> {
     println!("Hello from dhall mock project 👋");
 
     // Some Dhall data
     let data = r###"
         let Mock = ./dhall/Mock/package.dhall
-        in Mock.HttpMethod.GET
+        in Mock.HttpMethod.UNKNOW
```

Comment:
could you revert this change to keep the main in success?

Reply:
:+1: